<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1086</width>
    <height>907</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_camera">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>700</width>
      <height>600</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_time">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>20</y>
      <width>231</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_random_check">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>740</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>随机点名</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_supplement">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>300</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>漏签补签</string>
    </property>
   </widget>
   <widget class="QTableView" name="tableView_late">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>360</y>
      <width>211</width>
      <height>141</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_location">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>100</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>考勤地点</string>
    </property>
   </widget>
   <widget class="Line" name="line_1">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>690</y>
      <width>31</width>
      <height>131</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_supplement">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>300</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_log">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>690</y>
      <width>301</width>
      <height>161</height>
     </rect>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>833</y>
      <width>301</width>
      <height>16</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_blinks">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>770</y>
      <width>71</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>活体检测</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_start_check">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>770</y>
      <width>101</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>开始考勤</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_location_2">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>690</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>FaceNet (ResNet-SSD)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>FaceNet (MTCNN)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>RetinaFace</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ArcFace</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>CenterFace</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>DBFace</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>FaceBoxes</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>S3FD</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="bt_generator">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>760</y>
      <width>151</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>训练模型</string>
    </property>
   </widget>
   <widget class="QTableView" name="tableView_escape">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>530</y>
      <width>211</width>
      <height>141</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_load_model">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>720</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>加载模型</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_view">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>680</y>
      <width>211</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>查看结果</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_lcdName2">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>190</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>实到</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_class">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>70</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>考勤班级</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_time_hour">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>221</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcd_1">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>140</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_check_variation">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>760</y>
      <width>131</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>核验人脸数据库信息</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_listName2">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>510</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>迟到</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcd_2">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>180</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_open_camera">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>690</y>
      <width>181</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>打开相机</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_title">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>20</y>
      <width>400</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_logo">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>800</width>
      <height>600</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>800</width>
      <height>600</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>800</width>
      <height>600</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_leave">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>260</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>请假登记</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_check">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>140</y>
      <width>91</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>查询</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_location_2">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>230</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>设置考勤时间</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_exit">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>790</y>
      <width>211</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>退出系统</string>
    </property>
   </widget>
   <widget class="Line" name="line_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>821</width>
      <height>21</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_class">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>70</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>2020001</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2020002</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="lineEdit_leave">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>260</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_listName1">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>340</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>未到</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_gathering">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>690</y>
      <width>131</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>信息采集</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_lcdName1">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>150</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>应到</string>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>70</y>
      <width>20</width>
      <height>781</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_time_minute">
    <property name="geometry">
     <rect>
      <x>1011</x>
      <y>220</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_location">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>100</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>A</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>B</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>C</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>D</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>E</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1086</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSetting"/>
    <addaction name="actionExport"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
   <property name="whatsThis">
    <string>Author: datamonday</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSetting">
   <property name="text">
    <string>Setting</string>
   </property>
  </action>
  <action name="actionExport">
   <property name="text">
    <string>Export</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
